src/Tribe/I18n.php

```diff
@@ -187,6 +188,8 @@ protected function with_locale( $locale, callable $do, array $args = [] ) {
 
 		// Restore the `locale` filtering functions.
 		$wp_filter['locale'] = $locale_filters_backup;
+		// Restore the previous I10n object.
+		$l10n = $l10n_backup;
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $l10n

<sub>hash: ad348e7c9dbf1d3d064f40798bf8d546</sub>